.changeset/big-insects-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cultureamp/rich-text-toolkit": minor
```

Comment:
Technically this breaks previous usages of `listIsActive` method. Thoughts on the versioning?

Reply:
I think this is fine as a minor, given we're currently the only ones using this package.